app/src/main/java/org/fossasia/openevent/general/search/SearchLocationViewModel.kt

```diff
@@ -24,10 +34,42 @@ class SearchLocationViewModel(
     private val mutableShowShimmer = MutableLiveData<Boolean>()
     val showShimmer: LiveData<Boolean> = mutableShowShimmer
 
+    val placeSuggestions = MutableLiveData<List<CarmenFeature>>()
+
     fun saveSearch(query: String) {
         preference.putString(SAVED_LOCATION, query)
     }
 
+    private fun loadPlaceSuggestions(query: String) {
+        val geoCodingRequest = makeGeocodingRequest(query)
+
+        geoCodingRequest.enqueueCall(object : Callback<GeocodingResponse> {
```

Comment:
Cancel call `onCleared`

Reply:
Thank you  :) . I will do that

Reply:
Hello ,I would like to ask , I have called `loadPlaceSuggesstion` inside `compositeDisposable`. So why wouln't it cancel the call on dispose ?

Reply:
I have updated